src/import/import-compress.c

```diff
@@ -188,6 +214,39 @@ int import_uncompress(ImportCompress *c, const void *data, size_t size, ImportCo
                 break;
 #endif
 
+#if HAVE_ZSTD
+        case IMPORT_COMPRESS_ZSTD: {
+                ZSTD_inBuffer in = {
+                        .src = data,
+                        .size = size
+                };
+                ZSTD_outBuffer out = {
+                        .dst = newa(uint8_t, ZSTD_DStreamOutSize()),
+                        .size = ZSTD_DStreamOutSize()
+                };
+
+                while (in.pos < in.size) {
+                        size_t ret = ZSTD_decompressStream(c->zstd_d, &out, &in);
+                        if (ZSTD_isError(ret)) {
+                                return log_error_errno(
+                                        SYNTHETIC_ERRNO(EIO),
+                                        "Failed to decompress zstd stream: %s",
+                                        ZSTD_getErrorName(ret)
+                                );
```

Comment:
This is not very important, but we'd generally format those a bit differently:
```C
return log_error_errno(SYNTHETIC_ERRNO(EIO),
                       "Failed to decompress zstd stream: %s", ZSTD_getErrorName(ret));
```
The style with the args hanging below is not *wrong*, but it's quite different than the rest of the codebase.

Reply:
actually the style we use most often is breaking before the first arg, and then indenting to 16ch the next line. and the closing `)` we definitely put at the end of the last argument line, not on a separate line.

Reply:
Reformatted everything to 16ch line continuations, with no hanging closing parenthesis.